sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -332,19 +331,11 @@ private ServiceBusConnectionProcessor getOrCreateConnectionProcessor(MessageSeri
     }
 
     private ConnectionOptions getConnectionOptions() {
-        configuration = configuration == null ? Configuration.getGlobalConfiguration().clone() : configuration;
```

Comment:
The problem with removing this line is that if someone does:

```java
var client = new ServiceBusClientBuilder().configuration(null)
    .buildClient();
```

It won't use the default credential because you only assign it on object creation.

Reply:
Added the line back